Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Tooltips/ToolTip.cs

```diff
@@ -0,0 +1,540 @@
+        }
+
+        [SerializeField]
+        private bool showConnector = true;
+        /// <summary>
+        /// getter/setter for showing connecting stem between tooltip and parent object
+        /// </summary>
+        public bool ShowConnector
+        {
+            get
+            {
+                return showConnector;
+            }
+            set
+            {
+                showConnector = value;
+                //todo fix this
+
+                BaseMixedRealityLineDataProvider lineScript = GetComponent<BaseMixedRealityLineDataProvider>();
+                lineScript.enabled = value;
+            }
+        }
+
+        [SerializeField]
+        private DisplayModeType tipState;
+        /// <summary>
+        /// getter/setter for the display state of a tooltip
+        /// </summary>
+        public DisplayModeType TipState
+        {
+            get
+            {
+                return tipState;
+            }
+            set
+            {
+                tipState = value;
+            }
+        }
+
+        [SerializeField]
+        private DisplayModeType groupTipState;
+        /// <summary>
+        /// getter/setter for display state of group of tooltips
+        /// </summary>
+        public DisplayModeType GroupTipState
+        {
+            set
+            {
+                groupTipState = value;
+            }
+            get
+            {
+                return groupTipState;
+            }
+        }
+
+        [SerializeField]
+        private DisplayModeType masterTipState;
+        /// <summary>
+        /// getter/setter for display state of master tooltip
+        /// </summary>
+        public DisplayModeType MasterTipState
+        {
+            set
+            {
+                masterTipState = value;
+            }
+            get
+            {
+                return masterTipState;
+            }
+        }
+
+        [Tooltip("GameObject that the line and text are attached to")]
+        [SerializeField]
+        private GameObject anchor;
+        /// <summary>
+        /// getter/setter for ameObject that the line and text are attached to
+        /// </summary>
+        public GameObject Anchor
+        {
+            get
+            {
+                return anchor;
+            }
+            set
+            {
+                anchor = value;
+            }
+        }
+
+        [Tooltip("Pivot point that text will rotate around as well as the point where the Line will be rendered to.")]
+        [SerializeField]
+        private GameObject pivot;
+        /// <summary>
+        /// Pivot point that text will rotate around as well as the point where the Line will be rendered to. 
+        /// </summary>
+        public GameObject Pivot
+        {
+            get
+            {
+                return pivot;
+            }
+        }
+
+        [Tooltip("GameObject text that is displayed on the tooltip.")]
+        [SerializeField]
+        private GameObject label;
```

Comment:
any chance an app may need/want to access this?

Reply:
Easy enough to add property accessor as needed.